qemu/tests/cfg/blockdev_commit_reboot.cfg

```diff
@@ -0,0 +1,34 @@
+- blockdev_commit_reboot:
+    type = blockdev_commit_reboot
+    virt_test_type = qemu
+    only Linux
+    start_vm = yes
+    kill_vm = yes
+    storage_pools = default
+    storage_type_default = "directory"
+    storage_pool = default
+    image_size_data = 500M
```

Comment:
a data disk is required for this case ? I think we just make snapshots for system image, right?

Reply:
Have updated, thanks for pointing it out.